bin/ruby-build

```diff
@@ -1256,6 +1256,29 @@ build_package_ldflags_dirs() {
   done
 }
 
+build_package_enable_yjit() {
+  local rustc_installed rustc_ver
+
+  rustc_installed=""
+  if which rustc; then
+    rustc_ver=`rustc --version | cut -c7-10`
+    # Some kind of built-in bash comparison for dotted version strings would be awesome.
+    if [[ " ${rustc_ver} " == "1."[6789]* ]]; then
+      rustc_installed="1"
+    fi
+  fi
+
+  # If YJIT is explicitly specified, don't change anything
+  if [[ " ${RUBY_CONFIGURE_OPTS} " != *" --enable-yjit"* ]]; then
+    if [[ -z "${rustc_installed}" ]]; then
+      echo "rustc ${rustc_ver} is installed: enabling yjit by default"
```

Comment:
I think I would prefer that we say "building CRuby with YJIT" or "building CRuby without YJIT" instead of enabling/disabling, but open to arguments either way.

Reply:
Sounds good to me.

I think we should also mention how to disable in this message, since e.g. compilation might fail due to a badly setup/broken rust toolchain, and I think we wouldn't want users to open issues/discussions just to ask that.

Reply:
Okay. Updated the language.